client/lib/user/user.js

```diff
@@ -238,7 +238,28 @@ User.prototype.set = function( attributes ) {
 	return changed;
 };
 
+User.prototype.changeUser = function( username, password, callback ) {
+	if ( config.isEnabled( 'support-user' ) ) {
+		wpcom.changeUser( username, password, function( error ) {
+			if ( ! error ) {
+				this.fetch();
+			}
+			callback( error );
+		}.bind( this ) );
+	}
+};
+
+User.prototype.restoreUser = function() {
+	if ( config.isEnabled( 'support-user' ) ) {
+		wpcom.restoreUser();
+
+		this.fetch();
+	}
+}
```

Comment:
You're right the inspection came from jetbrains rather than the eslint config. I can investigate adding in the semi rule: http://eslint.org/docs/rules/semi#options